test/cookbooks/osl-rt-test/attributes/osl_request_teacker.rb

```diff
@@ -0,0 +1,70 @@
+default['osl-rt']['queues'].tap do |q|
+  q['Support'] = 'support'
+  q['Frontend Team'] = 'frontend'
+  q['Backend Team'] = 'backend'
+  q['DevOps Team'] = 'devops'
+  q['Marketing Team'] = 'advertising'
+  q['The Board Of Directors'] = 'board'
+end
+default['osl-rt']['db'].tap do |db|
+  db['type'] = 'mysql'
+  db['host'] = 'localhost'
+  db['name'] = 'rt'
+  db['username'] = 'rt-user'
+  db['password'] = 'rt-password'
+end
+
+default['osl-rt']['fqdn'] = 'example.org'
+default['osl-rt']['default'] = 'support'
+default['osl-rt']['root-password'] = 'my-epic-rt'
+default['osl-rt']['plugins'] = %w(RT::Extension::REST2 RT::Authen::Token)
+default['osl-rt']['lifecycles'] = {
+  'default' => {
+    'initial' => [ 'new' ],
+    'active' => [ 'open' ],
+    'inactive' => %w(stalled resolved rejected deleted),
+
+    'defaults' => {
+      'on_create' => 'new',
+      'on_merge' => 'resolved',
+      'approved' => 'open',
+      'denied' => 'rejected',
+    },
+
+    'transitions' => {
+      '' => %w(new open resolved),
+      'new' => %w(open stalled resolved rejected deleted),
+      'open' => %w(new stalled resolved rejected deleted),
+      'stalled' => %w(new open rejected resolved deleted),
+      'resolved' => %w(new open stalled rejected deleted),
+      'rejected' => %w(new open stalled resolved deleted),
+      'deleted' => %w(new open stalled rejected resolved),
+    },
+
+    'rights' => {
+      '* -> deleted' => 'DeleteTicket',
+      '* -> *' => 'ModifyTicket',
+    },
+    'actions' => {
+      'new -> open' => {
+        'label' => 'Open It',
+        'update' => 'Respond',
+      },
+    },
+  },
+  'new-wave' => {
+    'initial' => [ 'order in' ],
+    'active' => [ 'order work' ],
+    'inactive' => [ 'order up' ],
+
+    'transitions' => {
+      '' => [ 'order in' ],
+      'order in' => [ 'order work' ],
+      'order work' => [ 'order up' ],
+    },
+
+    'rights' => {
+      '* -> order in' => 'ResetOrder',
+    },
+  },
+}
```

Comment:
Sane defaults should be set in the primary cookbook attribute file, not the test cookbook. Any overrides should happen in the recipe as `node.default`.

Reply:
@zwall-bp I believe this still needs to be addressed.

Reply:
Please move all of these into the test recipe as `node.default` which is how we would normally do this in production.